### PR TITLE
Google Analytics

### PR DESCRIPTION
**Pull request by kalyanisjsu**

### CODE REVIEW

File: src/main/webapp/detailedReviews.html [hunk 1]
```diff
@@ -29,128 +40,123 @@
...
-                                    <li class="ratings-bar-style">
-                                        <div class="feature-wrap"><span>Resolution</span><strong class="pull-right">3.0</strong></div>
-                                        <div class="bar-wrap"><a style="width:60%;" class="votes-wrap"><span class="votes"></span></a></div>
-                                    </li>
-                                </ul>
-                            </div>
-                        </div>
-                    </div>
-                </div>
+            <!--<div class="row">
```
Comment (joejose2728) [inline src/main/webapp/detailedReviews.html]: Remove commented code

File: src/main/java/com/revify/controller/ReviewControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
+package com.revify.controller;
```
Comment (joejose2728) [inline src/main/java/com/revify/controller/ReviewControllerTest.java]: ReviewControllerTest should be in src/test/java/{packageName} folder

File: Test/ReviewControllerTest.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
+package com.revify.controller;
```
Comment (joejose2728) [inline Test/ReviewControllerTest.java]: This should be in src/test/java/{packageName} folder

**Comment by joejose2728** [comment]
@kalyanisjsu made some review comments. Please make the changes and commit again. Once done I will merge it.

**Comment by kalyanisjsu** [comment]
Done...

Thanks,
Kalyani Nirmal

**Pull request merged**